app/views/messages/ViewAllMessages.scala.html

```diff
@@ -54,17 +54,27 @@
              @messages("viewAllMessages.totalNumberOfMessages", request.messageStatistics.countOfNewMessages)
          }
 
-         @govukSelect(Select(
-             items = sortSelectItems,
-             name = "sortBy",
-             id = "sortBy",
-             label = Label(
-                 content = Text(messages("viewAllMessages.sort.label")),
-                 classes = "govuk-!-display-inline govuk-!-padding-right-2"
-             ),
-             classes = "govuk-!-display-inline"
-         ))
-     </div>
+        <form>
+            @govukSelect(Select(
+                items = sortSelectItems,
+                name = "sortBy",
+                id = "sortBy",
+                label = Label(
+                    content = Text(messages("viewAllMessages.sort.label")),
+                    classes = "govuk-!-display-inline govuk-!-padding-right-2"
+                ),
+                classes = "govuk-!-display-inline"
+            ))
+
+            @govukButton(
+                ButtonViewModel(Text(messages("viewAllMessages.sort.button")))
+                        .withCssClass("govuk-!-margin-left-2")
+                        .withId("sortBySubmit")
+            )
```

Comment:
Should the button be inline to the sort by dropdown?
![image](https://github.com/hmrc/emcs-tfe-frontend/assets/53828896/70c1cdfc-767a-43dd-bfc4-7a591981c292)

Reply:
I think there be a test in the ViewSpec also?

Reply:
Updated to align that button 
<img width="1459" alt="image" src="https://github.com/hmrc/emcs-tfe-frontend/assets/11269301/15ee8791-0560-4763-b632-b3c88fe9e15c">